corehq/apps/data_interfaces/views.py

```diff
@@ -155,6 +155,69 @@ def get_deleted_item_data(self, item_id):
         }
 
 
+class ArchiveFormView(DataInterfaceSection):
```

Comment:
toggle + Role, since the toggle is for testing, and the Role is for restrict access based on subscription.  we wouldn't want premium users to see a feature before it's gone through QA